style.css

```diff
@@ -601,11 +600,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
     background-size: 40%;
     padding-bottom: 3rem;
```

Comment:
- [ ] As requested by previous reviewer you shoould stick to the design as much as possible like color, font, spacing. please make sure that you adust the spacing as in the design. Please see image below for reference. Also adjust the size of list.
### Your design
![his](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/42d82f19-2b8d-41e5-a3ae-38fa91e1e02e)

### figma design
![fimga](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/056af3c8-aee4-4098-a829-56f2883c42ba)